nvchecker/core.py

```diff
@@ -371,7 +371,9 @@ def _process_result(r: RawResult) -> Union[Result, Exception]:
     return ValueError('no version returned')
 
 def check_version_update(
-  oldvers: VersData, r: Result,
+  oldvers: VersData,
+  r: Result,
+  log_noop: bool,
```

Comment:
We need a better name for it. Maybe `verbose` since verbosity is indeed increased.